### ISSUE TITLE
Load PNGs as a C++ resource instead of from disk.

### ISSUE THREAD

**Issue by gir489** [v0idp/AmongUsMenu#31]
The PNGs should be added as a resource to be compiled in to the DLL, then retrieved using something like FindResourceA.

**Comment by v0idp** [comment]
Yes we had this idea a while ago too. I will add this to the Development Project as a card and take a look later.
